quantum/quantum.c

```diff
@@ -345,6 +345,27 @@ bool process_record_quantum(keyrecord_t *record) {
             case ONESHOT_DISABLE:
                 oneshot_disable();
                 break;
+#endif
+#ifndef NO_COMPILE_KEYCODE
+#    if defined(QMK_KEYBOARD) && defined(QMK_KEYMAP)  // tests don't generate these defines
+            case QK_MAKE:                             // Compiles the firmware, and adds the flash command based on keyboard bootloader
+            {
+                uint8_t temp_mod = mod_config(get_mods()) | mod_config(get_oneshot_mods());
```

Comment:
Currently fails on xd002

`get_oneshot_mods` might not exist, the case when `NO_ACTION_ONESHOT` is enabled.